harvester_e2e_tests/integrations/test_upgrade.py

```diff
@@ -735,6 +767,8 @@ def test_verify_logging_pods(self, api_client):
         """ Verify logging pods and logs
         Criteria: https://github.com/harvester/tests/issues/535
         """
+        code, data = api_client.addons.get("cattle-logging-system/rancher-logging")
```

Comment:
You would need to check whether the cluster version is the versions with logging addon first.